sqlccl: TestShowCreateRedactableValues under race: statement timeout during DDL

pkg/ccl/testccl/sqlccl/sqlccl_test.TestShowCreateRedactableValues [failed](https://tanzanite.cluster.engflow.com/invocations/default/389e6840-3599-499a-b0e2-9cd00230a383?testReportRun=1&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [e2e88aa288c81282b20ca0640da983f97b234324](https://github.com/cockroachdb/cockroach/commits/e2e88aa288c81282b20ca0640da983f97b234324):


```
    show_create_test.go:81: CREATE TABLE table͉3 (col3_0 BIT(16), col̒3_1 INT4 NOT NULL, "\\U000BAB91col3_2" NAME NOT NULL, col3_3 REGTYPE NULL, col3_4 INT2 NOT NULL, col3_5 TIMESTAMPTZ NOT NULL, "%pcol3😴_6" FLOAT8 NOT NULL, col3_7 OID NULL, col3_8 BIT(6) NOT NULL, col3_9 GEOGRAPHY NOT NULL, col3_10 "char" NOT NULL, col3_11 INT4 NOT NULL, col3_12 INTERVAL NOT NULL, "col3\f_%p13" STRING NOT NULL AS (lower("\\U000BAB91col3_2")) VIRTUAL, "c\gol3_14" INT8 AS (col̒3_1 + col3_11) STORED, PRIMARY KEY (col3_8 ASC, col3_11 DESC), UNIQUE (col3_5 ASC) STORING (col̒3_1, "\\U000BAB91col3_2", col3_4, "%pcol3😴_6") WHERE ((table͉3.col3_5 <= '3000-01-01 00:00:00+00':::TIMESTAMPTZ) OR (table͉3."col3\f_%p13" >= e'allosaurus\U00002603':::STRING)) OR (table͉3."\\U000BAB91col3_2" != e'\U00002603':::STRING:::NAME), UNIQUE (col3_12 DESC) WHERE ((table͉3.col3_11 != 32767:::INT8) AND (table͉3.col3_10 <= 'allosaurus"':::STRING)) AND (table͉3.col3_5 = '2001-01-01 00:00:00+00 BC':::TIMESTAMPTZ), UNIQUE (col3_10, col3_3 ASC, ((col3_11 + col̒3_1) + col3_4)), FAMILY (col3_9), FAMILY (col3_3), FAMILY (col3_8, col3_12), FAMILY ("\\U000BAB91col3_2", col3_4), FAMILY ("c\gol3_14", col3_0), FAMILY (col3_11, col3_10, col̒3_1), FAMILY ("%pcol3😴_6", col3_5), FAMILY (col3_7))
    show_create_test.go:81: CREATE TABLE table4 (col4_0 TIMESTAMP NOT NULL, col4_1 REGTYPE, "co\fl4_2" BYTES NOT NULL, coļ4_3 INTERVAL NOT NULL, col4_4 STRING NOT NULL AS (CASE WHEN coļ4_3 IS NULL THEN e'allosaurusZ\x05Z&LF':::STRING ELSE e'allosaurus<71\x19p':::STRING END) STORED, PRIMARY KEY (col4_0, col4_4 ASC, "co\fl4_2" ASC))
    show_create_test.go:81: ALTER TABLE "table�1" INJECT STATISTICS e'[{"avg_size": 28, "columns": ["\\"cOl1 _1\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 105278669, "histo_buckets": [{"distinct_range": 0, "num_eq": 8663226510372343973, "num_range": 0, "upper_bound": "-27571"}, {"distinct_range": 6678851098536834000, "num_eq": 900, "num_range": 6678851098536833731, "upper_bound": "30802"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 2, "columns": ["\\"co\\"\\"l1_\'5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 694495474, "histo_buckets": [{"distinct_range": 0, "num_eq": 937157470212735959, "num_range": 0, "upper_bound": "694A81EC/5721D39"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 500000000, "upper_bound": "7A58B7F2/70FC682E"}, {"distinct_range": 0, "num_eq": 6696978089763550584, "num_range": 9000, "upper_bound": "7DF73F6B/A9E913EE"}, {"distinct_range": 136.93370592751612, "num_eq": 2722646237991310176, "num_range": 800, "upper_bound": "A33F7F93/D4948796"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 300000, "upper_bound": "B548689A/AB0C50A0"}, {"distinct_range": 5426456218867585000, "num_eq": 5764402756188760654, "num_range": 8032493844260765128, "upper_bound": "C0911816/87B5CBC"}, {"distinct_range": 2276746532145870600, "num_eq": 8517913418497137981, "num_range": 7600314818091920064, "upper_bound": "CAC7ABC0/F2BB8434"}, {"distinct_range": 5362200143792167000, "num_eq": 7507473973819574131, "num_range": 5362200143792166736, "upper_bound": "F18C1FA8/47048608"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 7906313455129665915, "upper_bound": "F4002190/BD0DCF68"}], "histo_col_type": "PG_LSN", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 22, "columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 366717466, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 10, "columns": ["col\u032F1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 198167576, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 8, "columns": ["\\"col 1_10\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 623809899, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "899363971.5585850527"}, {"distinct_range": 4932018062.875811, "num_eq": 6921238803353989731, "num_range": 5000000000, "upper_bound": "5.489676612823000442E+30"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 76741660, "row_count": 700000000}, {"avg_size": 14, "columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 578185020, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 12, "columns": ["\\"c\u0350ol1_\\\\v2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 317254625, "histo_col_type": "", "name": "__auto__", "null_count": 222860699, "row_count": 700000000}, {"avg_size": 17, "columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177500298, "histo_buckets": [{"distinct_range": 0, "num_eq": 2703995897838056794, "num_range": 0, "upper_bound": "GlUM3%``L"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 6, "columns": ["\\"co l1_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 297593393, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 87796600, "row_count": 700000000}, {"avg_size": 24, "columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 534565751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"avg_size": 11, "columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 588994590, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}]':::JSONB
    show_create_test.go:81: ALTER TABLE "tabLe2" INJECT STATISTICS e'[{"avg_size": 9, "columns": ["\\"c(o\\\\\\\\uE9D3l2_7\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 682843831628426515, "histo_col_type": "", "name": "__auto__", "null_count": 272118144854466393, "row_count": 1194275836189718240}, {"avg_size": 23, "columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 408825540669144432, "histo_col_type": "", "name": "__auto__", "null_count": 823816899147437660, "row_count": 1194275836189718240}, {"avg_size": 29, "columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432113674037582245, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 7292697370701081000, "num_eq": 40000000000, "num_range": 8713141941240636934, "upper_bound": "!"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 2653156045294102327, "upper_bound": "+"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 10000000000, "upper_bound": "D"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 4236178352245575628, "upper_bound": "I"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1194275836189718240}, {"avg_size": 24, "columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 879288825541335163, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "\\u0005"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 8799393322829629814, "upper_bound": "\\u0006"}, {"distinct_range": 6000, "num_eq": 1000000, "num_range": 6000, "upper_bound": "\\u0018"}, {"distinct_range": 0, "num_eq": 5818796853886675961, "num_range": 700, "upper_bound": "\\u001e"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 1111012330038351176, "row_count": 1194275836189718240}, {"avg_size": 19, "columns": ["\\"\U0001F606co(l\\"\\"2_5\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 740721208030090851, "histo_col_type": "", "name": "__auto__", "null_count": 306990089407337259, "row_count": 1194275836189718240}, {"avg_size": 24, "columns": ["\\"(col2_9\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1039203793541967016, "histo_col_type": "", "name": "__auto__", "null_count": 1026796510688977029, "row_count": 1194275836189718240}, {"avg_size": 14, "columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 293202783293246330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1194275836189718240}, {"avg_size": 10, "columns": ["\\"\'col2_4\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1189489446917621560, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1194275836189718240}, {"avg_size": 24, "columns": ["\\"\'col2_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 501712398641428782, "histo_col_type": "", "name": "__auto__", "null_count": 594801243698452249, "row_count": 1194275836189718240}, {"avg_size": 18, "columns": ["\\"c%aaol2_8\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 623401375893248696, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 156496445983876900, "num_eq": 2196427841088792827, "num_range": 512053260216218128, "upper_bound": "-2.1193828792652525"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5031537065759017632, "upper_bound": "-0.8701720912906605"}, {"distinct_range": 5686432014256942000, "num_eq": 400000000, "num_range": 6908975502372229293, "upper_bound": "-0.48835471831229005"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 4530341869728846677, "upper_bound": "0.24254019488909007"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 10000, "upper_bound": "0.40954233462224976"}, {"distinct_range": 679966539263568000, "num_eq": 2000000000, "num_range": 679966539263567954, "upper_bound": "1.5694230071032371"}, {"distinct_range": 6163719089254265000, "num_eq": 200, "num_range": 6163719089254264527, "upper_bound": "2.6478954435681454"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 1194275836189718240}]':::JSONB
    show_create_test.go:81: ALTER TABLE table͉3 INJECT STATISTICS e'[{"avg_size": 26, "columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2781783772437120253, "histo_col_type": "", "name": "__auto__", "null_count": 411367530959743212, "row_count": 4746387788206341324}, {"avg_size": 0, "columns": ["\\"\\\\\\\\U000BAB91col3_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1051104935972945007, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 29, "columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3777088570674686110, "histo_col_type": "", "name": "__auto__", "null_count": 1664623250877358616, "row_count": 4746387788206341324}, {"avg_size": 0, "columns": ["\\"col3\\\\f_%p13\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2176829450164861177, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 20, "columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183771179538813719, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 17, "columns": ["col\u03123_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2521267220862142655, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 3, "columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3200258358695305075, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 29, "columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3310166810263200882, "histo_col_type": "", "name": "__auto__", "null_count": 2080530642484124315, "row_count": 4746387788206341324}, {"avg_size": 27, "columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1630722394438261810, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 17, "columns": ["\\"%pcol3\U0001F634_6\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4496916094551176604, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 12, "columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 917978335279128084, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 1423408058115935700, "num_eq": 7676796755275272534, "num_range": 3802609115708269470, "upper_bound": "\'"}, {"distinct_range": 2021631450883339800, "num_eq": 2308616344850213902, "num_range": 4618091262512561059, "upper_bound": "?"}, {"distinct_range": 8830020833964443000, "num_eq": 20000000000, "num_range": 8830020833964442946, "upper_bound": "X"}, {"distinct_range": 3862051795432076300, "num_eq": 3396496543902411928, "num_range": 5902169638776942982, "upper_bound": "Z"}, {"distinct_range": 8912379233865771000, "num_eq": 0, "num_range": 8912379233865771382, "upper_bound": "f"}, {"distinct_range": 0, "num_eq": 5039961815067787669, "num_range": 0, "upper_bound": "\U00002603"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 1, "columns": ["\\"c\\\\gol3_14\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4001513389597524116, "histo_col_type": "", "name": "__auto__", "null_count": 571334953508043299, "row_count": 4746387788206341324}, {"avg_size": 28, "columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4158982999803119783, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 21, "columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1282959539584939244, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "001001"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 6906271956447613220, "upper_bound": "001101"}, {"distinct_range": 428199108907302900, "num_eq": 7849823933369119280, "num_range": 428199108907302927, "upper_bound": "011100"}, {"distinct_range": 16097842.236610213, "num_eq": 100000000000, "num_range": 30000000, "upper_bound": "011111"}, {"distinct_range": 9153545374775467000, "num_eq": 50, "num_range": 9153545374775466753, "upper_bound": "111001"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}, {"avg_size": 8, "columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2859010113963223563, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4746387788206341324}]':::JSONB
    show_create_test.go:81: ALTER TABLE table4 INJECT STATISTICS e'[{"avg_size": 26, "columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3060312961175086697, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 3284512402591522915}, {"avg_size": 28, "columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2084872439829075994, "histo_col_type": "", "name": "__auto__", "null_count": 334336010049298042, "row_count": 3284512402591522915}, {"avg_size": 20, "columns": ["\\"co\\\\fl4_2\\""], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1639442434001278498, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3284512402591522915}, {"avg_size": 20, "columns": ["co\u013C4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1928533187264166327, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3284512402591522915}, {"avg_size": 13, "columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1632719127614566167, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3284512402591522915}]':::JSONB
    show_create_test.go:81: CREATE TYPE rand_typ_0 AS ENUM ('bao')
    show_create_test.go:81: CREATE TYPE rand_typ_1 AS ENUM ('me', 'z', 'ag', 'ntneie', 'ytb')
    show_create_test.go:81: CREATE TYPE rand_typ_2 AS ENUM ('jlp', 'qpooc')
    show_create_test.go:81: CREATE TYPE rand_typ_3 AS ENUM ('ba', 'ukxvt', 'by', 'iszxcb')
    show_create_test.go:81: CREATE TYPE rand_typ_4 AS ENUM ('n', 'k', 'a')
    show_create_test.go:81: SET statement_timeout = '5s';
    show_create_test.go:103: ALTER TABLE defaultdb.public.table4
        	ADD COLUMN
        		"c �ol1"
        			STRING COLLATE da_DK
        			AS ((e'\U0004EE6D\U0001D2EA\U000579F1\U00077C6E\U000C744F\U000C0F6B' COLLATE da_DK)) STORED
        			NOT NULL
        			CHECK (NULL)
    show_create_test.go:103: CREATE TABLE defaultdb.public.t😚ab4 (
        	col0_0
        		PG_LSN,
        	col0_1
        		FLOAT4 NOT NULL,
        	col0_2
        		FLOAT8 NOT NULL,
        	"c%qol0_3"
        		BIT(13),
        	"co̿l0_%p4"
        		REGTYPE,
        	"col0""_5"
        		FLOAT8 AS (col0_1 + 0.1325482577085495:::FLOAT8) STORED NOT NULL,
        	col0_6
        		FLOAT8 AS (col0_1 + col0_2) VIRTUAL NULL,
        	"\\x8acol0_7"
        		FLOAT8 AS (col0_2 + NULL) VIRTUAL NULL,
        	PRIMARY KEY (col0_2),
        	INDEX ("\\x8acol0_7", "co̿l0_%p4" DESC),
        	FAMILY ("c%qol0_3", col0_1),
        	FAMILY ("col0""_5"),
        	FAMILY (col0_0),
        	FAMILY ("co̿l0_%p4", col0_2)
        )
    show_create_test.go:103: ALTER TABLE defaultdb.public.t😚ab4 ADD COLUMN "coL2" TIME DEFAULT ('1972-07-14 21:42:50.000875':::TIMESTAMP) NULL
    show_create_test.go:103: CREATE UNIQUE INDEX idx1 ON defaultdb.public.t😚ab4 ("c%qol0_3" DESC) STORING ("co̿l0_%p4", col0_1)
    show_create_test.go:103: CREATE SCHEMA "schema\\u1AA41"
    show_create_test.go:120: pq: query execution canceled due to statement timeout
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestShowCreateRedactableValues1251933528
--- FAIL: TestShowCreateRedactableValues (71.17s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestShowCreateRedactableValues.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34815

pkg/ccl/testccl/sqlccl/sqlccl_test.TestShowCreateRedactableValues [failed](https://tanzanite.cluster.engflow.com/invocations/default/9e314f26-e389-4c63-9066-d75bf5b1bb33?testReportRun=2&testReportShard=8&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [53e4efd389167110d687e5dea8bf915fe2582a25](https://github.com/cockroachdb/cockroach/commits/53e4efd389167110d687e5dea8bf915fe2582a25):


```
        						('3':::STRING:::NAME,),
        						(e')[\x0e\t\x18':::STRING:::NAME,),
        						(e'P\x1de\x16Z':::STRING:::NAME,)
        					),
        				PARTITION "|0_part2"
        					VALUES IN (
        						(e'\x0bV/\x10H':::STRING:::NAME,),
        						(e'\x15{nu\x06\x1a\x14;':::STRING:::NAME,),
        						(e'Jw"\n"X"':::STRING:::NAME,),
        						(e'\x1eQ':::STRING:::NAME,),
        						(e'*h\x06':::STRING:::NAME,),
        						('l5':::STRING:::NAME,),
        						(e':?;-#\x07\x1a\x1c\x01':::STRING:::NAME,),
        						(e'\x15YL{fH':::STRING:::NAME,)
        					),
        				PARTITION "0_parT\\xd43"
        					VALUES IN (
        						('.h*^Xm ':::STRING:::NAME,),
        						(e'CEV*lp\rux':::STRING:::NAME,),
        						('c-)":xy':::STRING:::NAME,),
        						('D':::STRING:::NAME,),
        						('"':::STRING:::NAME,),
        						(e'ic4V\'\x063\x04o':::STRING:::NAME,),
        						(e'<\rX|\x16z':::STRING:::NAME,),
        						('8':::STRING:::NAME,)
        					),
        				PARTITION "0_part4"
        					VALUES IN (
        						(e'A~\r/O':::STRING:::NAME,),
        						(e'\x1d':::STRING:::NAME,),
        						('m-IGX)@S':::STRING:::NAME,),
        						(e'Z\x11}':::STRING:::NAME,),
        						('/,':::STRING:::NAME,),
        						(e'I\x1eOX':::STRING:::NAME,),
        						(e'\x18\'E[':::STRING:::NAME,),
        						(e'\x1c\t5':::STRING:::NAME,),
        						('O':::STRING:::NAME,)
        					)
        			),
        	INDEX (lower("%pco😧l0_2") ASC, col0_3) STORING(col0_0),
        	UNIQUE (col0_3 DESC, "%pco😧l0_2" DESC),
        	INDEX (col0_3, "%pco😧l0_2" DESC) STORING("col0'_4"),
        	UNIQUE ("col0'_4" DESC, col0_0 DESC),
        	UNIQUE (col0_0) STORING(" col0_1", "%pco😧l0_2", col0_3)
        )
    show_create_test.go:103: CREATE INDEX idx2 ON defaultdb.public.table2 ("c)ol2_3") STORING (col2_9)  VISIBILITY 0.03
    show_create_test.go:120: pq: query execution canceled due to statement timeout
    panic.go:541: -- test log scope end --
test logs left over in: outputs.zip/logTestShowCreateRedactableValues2370032559
--- FAIL: TestShowCreateRedactableValues (69.60s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=8</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestShowCreateRedactableValues.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/testccl/sqlccl/show_create_test.go
+++ b/pkg/ccl/testccl/sqlccl/show_create_test.go
@@ -76,7 +76,7 @@ func TestShowCreateRedactableValues(t *testing.T) {
 
 	// Perform a few random initial CREATE TABLEs and check for PII leaks.
 	setup := sqlsmith.RandTablesPrefixStringConsts(rng, pii)
-	setup = append(setup, "SET statement_timeout = '5s';")
+	setup = append(setup, "SET statement_timeout = '30s';")
 	for _, stmt := range setup {
 		t.Log(stmt)
 		if _, err := sqlDB.ExecContext(ctx, stmt); err != nil {

